lib/sync/sync-scheduler.js

```diff
@@ -43,20 +43,19 @@ SyncScheduler.prototype.checkDatasetsForSyncing = function(cb) {
     }
 
     var datasetClientsToSync = [];
-    var datasetClientsToRemove = [];
 
     _.each(datasetClients, function(datasetClientJson){
       var datasetClient = DatasetClient.fromJSON(datasetClientJson);
       var datasetId = datasetClient.getDatasetId();
-      if (datasetClient.shouldSync()) {
+      if (datasetClient.shouldDeactiveSync()) {
+        //if the a datasetclient is not active, we don't need to sync it
+        syncUtil.doLog(datasetId, 'info', 'Deactivating sync for client ' + datasetId + '. No client instances have accessed in ' + datasetClient.getConfig().clientSyncTimeout * 1000 + 'ms');
```

Comment:
Can we drop this log level to `debug`?
See https://issues.jboss.org/browse/FH-3220 for more info.
Hoping to have something in the README with the reasons why we'll only log at `debug`